components/ip2location/actions/query-ip-info/query-ip-info.mjs

```diff
@@ -0,0 +1,57 @@
+import { axios } from "@pipedream/platform";
+
+export default {
+  name: "Query IP address",
+  description: "Query IP address using IP2Location API.",
+  key: "ip2location-query-ip-info",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    ipl_api_key: {
+      type: "app",
+      app: "ip2location",
+      description: "You can sign up for a trial key at [here](https://www.ip2location.com/register?id=1005).",
+    },
+    ip_address: {
+      type: "string",
+      label: "IP Address",
+      description: "IP address (IPv4 or IPv6) for reverse IP location lookup purposes.",
+    },
+    package: {
+      type: "string",
+      label: "Package",
+      description: "Web service package of different granularity of return information. Please refer to the pricing table in our [documentation](https://www.ip2location.com/web-service/ip2location) for the information returned. Valid value: `WS1` | `WS2` | `WS3` | `WS4` | `WS5` | `WS6` | `WS7` | `WS8` | `WS9` | `WS10` | `WS11` | `WS12` | `WS13` | `WS14` | `WS15` | `WS16` | `WS17` | `WS18` | `WS19` | `WS20` | `WS21` | `WS22` | `WS23` | `WS24` | `WS25`",
```

Comment:
It would be great if you have an options constant property with an array of strings to choose from using the UI eg.

```javascript
   ...
   description: "Web service package of different granularity of return information. Please refer to the pricing table in our [documentation](https://www.ip2location.com/web-service/ip2location) for the information returned.",
   options: constants.PACKAGE_OPTIONS,
```

where you can assign the values in a file named like this:
- **ip2location**
  - **common**
    - *constants.mjs*

and the content of `constants.mjs` will be like this:
```javascript
const PACKAGE_OPTIONS = [
  "WS1",
  "WS2",
  "WS3",
  "WS4",
  "WS5",
  "WS6",
  "WS7",
  "WS8",
  "WS9",
  "WS10",
  "WS11",
  "WS12",
  "WS13",
  "WS14",
  "WS15",
  "WS16",
  "WS17",
  "WS18",
  "WS19",
  "WS20",
  "WS21",
  "WS22",
  "WS23",
  "WS24",
  "WS25",
];

export default {
  PACKAGE_OPTIONS,
};
```